packages/react-components/src/templates/DiscoverPageBody.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ComponentProps } from 'react';
+import css from '@emotion/css';
+
+import { PagesSection, RichText, MembersSection } from '../organisms';
+import { perRem } from '../pixels';
+import { Display, Card, Paragraph, Link } from '../atoms';
+import { CtaCard } from '../molecules';
+
+const styles = css({
+  display: 'grid',
+  gridRowGap: `${72 / perRem}em`,
+  marginBottom: `${24 / perRem}em`,
```

Comment:
I think we usually use `padding` for those (margins are usually not what you want regarding collapsing, background, ...)

Reply:
idk if there's a specific reason here, otherwise
```suggestion
  paddingBottom: `${24 / perRem}em`,
```